contracts/src/v0.8/KeeperRegistry.sol

```diff
@@ -549,7 +549,12 @@ contract KeeperRegistry is
    */
   function getActiveUpkeepIDs(uint256 startIndex, uint256 maxCount) external view override returns (uint256[] memory) {
     uint256 maxIdx = s_upkeepIDs.length();
-    if (startIndex >= maxIdx) revert IndexOutOfRange();
+    if (startIndex == maxIdx) {
+      return new uint256[](0);
```

Comment:
this feels very hacky to me, I would rather have edge case logic on the node

Reply:
Done